CHANGELOG-1.0.md

```diff
@@ -0,0 +1,60 @@
+# Changelog since v1.0.1
+
+## Notable Changes
+
+- Cherry picks PR #172: Added extra verification of source PersistentVolumeClaim before creating snapshot.([#175](https://github.com/kubernetes-csi/external-snapshotter/pull/175), [@xing-yang](https://github.com/xing-yang))
+
+# Changelog since v0.4.1
+
+## Breaking Changes
+* c422fbd Switch to use TypedLocalObjectReference in core API.
```

Comment:
In the preview, these don't show up as links. We may need to manually substitute the PR link for these.

Reply:
Ok, I'll look them up.